docs/monitors/collectd-activemq.md

```diff
@@ -23,9 +23,9 @@ Monitor Type: `collectd/activemq`
 | `name` | no | `string` |  |
 | `serviceName` | no | `string` | This is how the service type is identified in the SignalFx UI so that you can get built-in content for it.  For custom JMX integrations, it can be set to whatever you like and metrics will get the special property `sf_hostHasService` set to this value. |
 | `serviceURL` | no | `string` | The JMX connection string.  This is rendered as a Go template and has access to the other values in this config. NOTE: under normal circumstances it is not advised to set this string directly - setting the host and port as specified above is preferred. (**default:** `service:jmx:rmi:///jndi/rmi://{{.Host}}:{{.Port}}/jmxrmi`) |
-| `instancePrefix` | no | `string` |  |
-| `username` | no | `string` |  |
-| `password` | no | `string` |  |
+| `instancePrefix` | no | `string` | Prefixes the generated plugin instance with prefix. If a second InstancePrefix is specified in a referenced MBean block, the prefix specified in the Connection block will appear at the beginning of the plugin instance, the prefix specified in the MBean block will be appended to it |
```

Comment:
Make the option names start with lower case since that is what users will see.

Reply:
Also put them in backquotes.

Reply:
added